projects/ngrx.io/src/karma.conf.js

```diff
@@ -25,14 +25,8 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['CustomChrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Required, otherwise the CI needs to install chromium (which we did in CircleCI via an orb)